apps/openssl.cnf

```diff
@@ -1,7 +1,9 @@
 #
 # OpenSSL example configuration file.
-# This is mostly being used for generation of certificate requests.
+# See doc/man5/config.pod for more info.
 #
+# This is mostly being used for generation of certificate requests,
```

Comment:
Is this really true?  Lots of TS stuff, for example.

Reply:
I am open to suggestions.. tsa is mainly certificate related too right?

Reply:
not a big deal, maybe just delete the comment.  or leave as-is.

Reply:
I think I will leave it as is for now.